packages/web-components/src/input/README.md

```diff
@@ -0,0 +1,177 @@
+- `aria-busy`
+  - Indicates an element is being modified and that assistive technologies may want to wait until the changes are complete before informing the user about the update.
+- `aria-controls`
+  - Identifies the element (or elements) whose contents or presence are controlled by the element on which this attribute is set.
+- `aria-current`
+  - A non-null `aria-current` state on an element indicates that this element represents the current item within a container or set of related elements.
+- `aria-describedby`
+  - Identifies the element (or elements) that describes the element on which the attribute is set.
+- `aria-details`
+  - The global `aria-details` attribute identifies the element (or elements) that provide additional information related to the object.
+- `aria-disabled`
+  - Indicates that the element is perceivable but disabled, so it is not editable or otherwise operable.
+- `aria-errormessage`
+  - Identifies the element that provides an error message for that object.
+- `aria-flowto`
+  - Identifies the next element (or elements) in an alternate reading order of content. This allows assistive technology to override the general default of reading in document source order at the user's discretion.
+- `aria-haspopup`
+  - Indicates the availability and type of interactive popup element that can be triggered by the element on which the attribute is set.
+- `aria-hidden`
+  - Indicates whether the element is exposed to an accessibility API.
+- `aria-invalid`
+  - Indicates the entered value does not conform to the format expected by the application.
+- `aria-keyshortcuts`
+  - Indicates keyboard shortcuts that an author has implemented to activate or give focus to an element.
+- `aria-label`
+  - Defines a string value that labels an interactive element.
+- `aria-labelledby`
+  - Identifies the element (or elements) that labels the element it is applied to.
+- `aria-live`
+  - Indicates that an element will be updated, and describes the types of updates the user agents, assistive technologies, and user can expect from the live region.
+- `aria-owns`
+  - Identifies an element (or elements) in order to define a visual, functional, or contextual relationship between a parent and its child elements when the DOM hierarchy cannot be used to represent the relationship.
+- `aria-relevant`
+  - Indicates what notifications the user agent will trigger when the accessibility tree within a live region is modified.
+- `aria-roledescription`
+  - Defines a human-readable, author-localized description for the role of an element.
+
+<br />
+<hr />
+<br />
+
+## **Preparation**
+
+<br />
+
+### **Fluent Web Component v3 v.s Fluent React 9**
+
+<br />
+
+**Component and Slot Mapping**
+
+| Fluent UI React 9 | Fluent Web Components 3 |
+| ----------------- | ----------------------- |
+| `<Input>`         | `<fluent-input>`        |
```

Comment:
Yeah, completely agree with you here. I had similar concerns about the name <fluent-input> and whether it might be confusing considering the standard HTML `<input>` element. I was initially prioritizing React parity, but I think it's probably more important to choose a name that accurately reflects the component's purpose and features, and `<fluent-text-input>` might be more appropriate